content/en/docs/concepts/services-networking/endpoint-slices.md

```diff
@@ -48,17 +48,17 @@ metadata:
     kubernetes.io/service-name: example
 addressType: IPv4
 ports:
-  - name: http
-    protocol: TCP
-    port: 80
+- name: http
+  protocol: TCP
+  port: 80
```

Comment:
IMO, the extra indentation helps people see which fields are part of which other fields.